recipes-connectivity/prplmesh/prplmesh.bb

```diff
@@ -8,27 +8,123 @@ inherit cmake pythonnative
 S = "${WORKDIR}/git"
 OECMAKE_SOURCEPATH = "${S}/"
 
+SUMMARY = "prplMesh"
 SECTION = "net"
 LICENSE = "BSD-2-Clause-Patent.txt"
```

Comment:
It's uncommon practice to have `.txt` in name of the license. Is it correct?

Reply:
@arnout add license file with `.txt` extension when created `meta-prplmesh` repo.
I'll ask him allow to rename license file

Reply:
The files in the LICENSES directory of prplMesh were created with the REUSE tool, so I guess they're correctly named...

Reply:
REUSE tool is not Yocto-style one. Common Yocto solution is the following: 

1. Name reference license file as in SPDX short identifier. Without any extensions. e.g. as in https://github.com/openembedded/openembedded-core/tree/master/meta/files/common-licenses
2. If there-is some well-known non-SPDX names for license files then aliases added with `SPDXLICENSEMAP` variable flags as in https://github.com/openembedded/openembedded-core/blob/master/meta/conf/licenses.conf

So I highly recommend to rename this file and remove `.txt` from name

Reply:
We're not going to rename the file in prplmesh just because yocto has a different convention.

However, mainly you mean that we don't need to use the license file from the prplmesh project, but can instead use one from the common licenses?

Reply:
Solved in #17